sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/implementation/converters/CommunicationIdentifierConverter.java

```diff
@@ -3,68 +3,39 @@
 
 package com.azure.communication.rooms.implementation.converters;
```

Comment:
I dont think this is being used anywhere anymore? Please double check and remove if true.